frame/ranked-collective/src/lib.rs

```diff
@@ -490,7 +497,7 @@ pub mod pallet {
 		#[pallet::call_index(0)]
 		#[pallet::weight(T::WeightInfo::add_member())]
 		pub fn add_member(origin: OriginFor<T>, who: AccountIdLookupOf<T>) -> DispatchResult {
-			let _ = T::PromoteOrigin::ensure_origin(origin)?;
+			T::AddOrigin::ensure_origin(origin)?;
```

Comment:
Function docs needs to be updated.

Reply:
Done. + Refresh docs to some other functions